cpp/csp/python/PyStructToJson.cpp

```diff
@@ -220,9 +246,7 @@ rapidjson::Value pyDictKeyToName( PyObject * py_key, rapidjson::Document& doc )
         auto json_obj = doubleToJson( key, doc );
         if ( json_obj.IsNull() )
         {
-            auto * str_obj = PyObject_Str( py_key );
-            Py_ssize_t len = 0;
-            const char * str = PyUnicode_AsUTF8AndSize( str_obj, &len );
+            auto[str, len] = pyObjectToString( py_key );
```

Comment:
Just use `PyObjectPtr::own( PyObject_Str( py_key ) )` here, no need to call the helper method